src/commons/temporal.h

```diff
@@ -27,4 +29,86 @@
 	*/
 	char *temporal_get_string_time(const char* format);
 
+	/**
+	* @NAME: t_state
+	* @DESC: Estado de una variable temporal.
+	*/
+	typedef enum {
+		STOPPED,
+		RUNNING
+	} t_state;
+
+	/**
+	* @NAME: t_temporal
+	* @DESC: Estructura de una Variable temporal.
+	*/
+	typedef struct {
+		struct timespec* current;
+		int64_t elapsed_ms;
+		t_state state;
+	} t_temporal;
+
+	/**
+	* @NAME: temporal_create
+	* @DESC: Crea una variable temporal.
```

Comment:
Aclaro en la descripción que el cronómetro ya está iniciado.
```suggestion
	* @DESC: Crea una variable temporal e inicia su cronómetro.
```